e2e_test/test/e2e_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('setup test', () {
```

Comment:
part of the reason we set it up without package:test originally is that these test are explicitly not hermetic and are order dependent.

What is the advantage of using package:test here?
Is there any concept of 'before test suite' or similar that we could use for this?